Charts/eiger/deployment/odin-server.cfg

```diff
@@ -0,0 +1,28 @@
+[server]
+debug_mode  = 0
+http_port   = 8888
+http_addr   = 0.0.0.0
+adapters    = fp, fr, mw, ef
+
+[tornado]
+logging = error
+
+[adapter.fp]
+module = odin_data.control.frame_processor_adapter.FrameProcessorAdapter
+endpoints = {{ $sep := "" }}{{- range $idx, $e := until (int $.Values.odinDataCount) }}{{ $sep }}{{ $.Values.detectorName }}-odin-data-{{ $idx }}:10004{{ $sep = ", " }}{{- end }}
```

Comment:
would you be able to use this to make the above more legible? https://stackoverflow.com/a/42724991

Reply:
Hmmm, You would need some kind of list comprehension to feed into the Join function and I'm not sure there is a way to do that.